cmd/kapacitord/main.go

```diff
@@ -89,33 +89,42 @@ func (m *Main) Run(args ...string) error {
 		}
 
 		signalCh := make(chan os.Signal, 1)
-		signal.Notify(signalCh, os.Interrupt, syscall.SIGTERM)
-		reloadSignal := make(chan os.Signal, 1)
-		signal.Notify(reloadSignal, os.Interrupt, syscall.SIGHUP)
+		signal.Notify(signalCh, os.Interrupt, syscall.SIGTERM, syscall.SIGHUP)
 		m.Logger.Println("I! Listening for signals")
 
 		// Block until one of the signals above is received
 	Loop:
 		for {
 			select {
-			case <-signalCh:
-				m.Logger.Println("I! Signal received, initializing clean shutdown...")
-				go func() {
-					cmd.Close()
-				}()
-				break Loop
-			case <-reloadSignal:
-				m.Logger.Println("I! SIGHUP received, Reloading tasks/templates/handlers directory...")
-				if err := cmd.Server.LoadService.Load(); err != nil {
-					m.Logger.Println(fmt.Sprintf("E! Failed to reload tasks/templates/handlers: %s", err))
-
-					if _, ok := err.(load.HardError); ok {
-						go func() {
-							cmd.Close()
-						}()
-						break Loop
+
+			case s := <-signalCh:
```

Comment:
@nathanielc there has to be a better way than this, but I couldn't find an example. This could be my end-of-day brain not thinking straight. Thoughts?